yaml-tests/src/test/resources/disabled-planner-rewrites/in-predicate.yamsql

```diff
@@ -61,33 +62,50 @@ test_block:
       - query: select a, b from ta where b in (1 + 0, 3 + 0, 5, 7)
       - unorderedResult: [{A: 2, 7}, {A: 4, 5}, {A: 6, 3}, {A: 8, 1}]
     -
-      # TODO TODO (IN expansion in cascades planner should distinct the in-list to avoid duplicates)
       # LONG value matched against IN list of repeated LONG values
       - query: select a, b from ta where b in (1, 1, 1, 1)
+      - explain: "ISCAN(F1 <,>) | FLATMAP q0 -> { EXPLODE arrayDistinct(promote(@c10 AS ARRAY(LONG))) | FILTER q0.B EQUALS _ AS q1 RETURN (q0.A AS A, q0.B AS B) }"
+      - initialVersionLessThan: !current_version
       - result: [{A: 8, 1}, {A: 8, 1}, {A: 8, 1}, {A: 8, 1}]
+      # IN expansion in cascades planner distincts the in-list since this version preventing duplicate results with nested loop joins
+      - initialVersionAtLeast: !current_version
+      - result: [{A: 8, 1}]
     -
-      # TODO TODO (IN expansion in cascades planner should distinct the in-list to avoid duplicates)
       # LONG value matched against IN list of repeated LONG values
       - query: select a, b from ta where b in (1, 2, 1, 3)
+      - explain: "ISCAN(F1 <,>) | FLATMAP q0 -> { EXPLODE arrayDistinct(promote(@c10 AS ARRAY(LONG))) | FILTER q0.B EQUALS _ AS q1 RETURN (q0.A AS A, q0.B AS B) }"
+      - initialVersionLessThan: !current_version
       - unorderedResult: [{A: 6, 3}, {A: 7, 2}, {A: 8, 1}, {A: 8, 1}]
+      # IN expansion in cascades planner distincts the in-list since this version preventing duplicate results with nested loop joins
```

Comment:
You don't have to leave a comment why a thing now works the way it should have worked from the beginning. Maybe you can rewrite these comments in a way that they explain why the old results are in fact the wrong results, instead of explaining why the new result is the new result :-)